cupy/_statistics/order.py

```diff
@@ -288,29 +294,33 @@ def percentile(a, q, axis=None, out=None, interpolation='linear',
         axis (int or tuple of ints): Along which axis or axes to compute the
             percentiles. The flattened array is used by default.
         out (cupy.ndarray): Output array.
-        interpolation (str): Interpolation method when a quantile lies between
+        method (str): Interpolation method when a quantile lies between
             two data points. ``linear`` interpolation is used by default.
             Supported interpolations are``lower``, ``higher``, ``midpoint``,
             ``nearest`` and ``linear``.
         keepdims (bool): If ``True``, the axis is remained as an axis of
             size one.
+        interpolation (str): Deprecated name for the method keyword argument.
 
     Returns:
         cupy.ndarray: The percentiles of ``a``, along the axis if specified.
 
     .. seealso:: :func:`numpy.percentile`
     """
+    if interpolation is not None:
+        method = _check_interpolation_as_method(
+            method, interpolation, 'percentile')
     if not isinstance(q, cupy.ndarray):
         q = cupy.asarray(q, dtype='d')
     q = cupy.true_divide(q, 100)
     if not _quantile_is_valid(q):  # synchronize
         raise ValueError('Percentiles must be in the range [0, 100]')
-    return _quantile_unchecked(a, q, axis=axis, out=out,
-                               interpolation=interpolation, keepdims=keepdims)
+    return _quantile_unchecked(
+        a, q, axis=axis, out=out, method=method, keepdims=keepdims)
 
 
-def quantile(a, q, axis=None, out=None, interpolation='linear',
-             keepdims=False):
+def quantile(a, q, axis=None, out=None, method='linear', keepdims=False, *,
+             interpolation=None):
```

Comment:
By the way, the `overwrite_input` argument is missing.  Until CuPy supports it, the signature could be `a, q, axis=None, out=None, *, ...`.

It seems OK to simply ignore `overwrite_input` (maybe with a performance warning).